warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -1,6 +1,7 @@
 package datawave.query.jexl.functions;
 
 import com.google.common.base.CharMatcher;
+import com.google.common.base.Function;
```

Comment:
This might be an unused import.  I don't see Function referenced.

Reply:
Un-used imports removed.  Thanks!